testing/Readme.md

```diff
@@ -55,3 +56,33 @@ Flags:
   - enable: deletes the DENYINTERNET/DENYLOADBALANCER rules
   - NSG: sets NSG to a custom NSG if desired [NSG1 default]
 
+## Install LME on the cluster:
+### InstallTestbed.ps1
+## Usage
+| **Parameter**     | **Alias** | **Description**                                                                        | **Required** |
+|-------------------|-----------|----------------------------------------------------------------------------------------|--------------|
+| $ResourceGroup    | -g        | The name of the resource group that will be created for storing all testbed resources. | Yes          |
+| $NumClients       | -n        | The number of Windows clients you have created; defaults to 2                          | No           |
+| $DomainController | -w        | The name of the domain controller in the cluster; defaults to "DC1"                    | No           |
+| $LinuxVm          | -l        | The name of the linux server in the cluster; defaults to "LS1"                         | No           |
+
+Example:
+```
+./InstallTestbed.ps1 --ResourceGroup YourResourceGroup 
```

Comment:
Just one dash for PowerShell arguments. I can't say I fully understand the behavior, but if you run `./InstallTestbed.ps1 --ResourceGroup YourResourceGroup`, `$ResourceGroup` is somehow saved as `-ResourceGroup`
```suggestion
./InstallTestbed.ps1 -ResourceGroup YourResourceGroup 
```

Reply:
Done